test/fixtures/transform_params.html

```diff
@@ -0,0 +1,3 @@
+<span>{{ content.string | repeat:4 }}</span>
+<span>{{ content.string | replace:'B':'C' }}</span>
```

Comment:
your point about the arguments is very convincing. Let's leave them as they are. As for the last comment. I think that is fair to say. It seems to fit into the the pattern. I was never really super satisfied with the syntax I came up for this `_foreach="item:list"` but now we have to stick with it. I think it's still fair. It makes sense  I think